iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepGrammar.java

```diff
@@ -170,10 +170,15 @@ public ObjectExpression OBJECT_EXPRESSION() {
 
   public Property PROPERTY() {
     return b.<Property>nonterminal(BicepLexicalGrammar.PROPERTY).is(
-      f.objectProperty(
-        IDENTIFIER(),
-        b.token(Punctuator.COLON),
-        EXPRESSION()));
+      b.firstOf(
+        f.objectProperty(
+          IDENTIFIER(),
+          b.token(Punctuator.COLON),
+          EXPRESSION()),
+        f.objectProperty(
+          INTERPOLATED_STRING_TYPE(),
+          b.token(Punctuator.COLON),
+          EXPRESSION())));
```

Comment:
With my suggestion from below this would reduce to:
```
f.objectProperty(
    b.firstOf(IDENTIFIER(), INTERPOLATED_STRING_TYPE()),
    b.token(Punctuator.COLON),
    EXPRESSION())
```